### PR TITLE
Optimization fixes

### PR DESCRIPTION
**Pull request by Onyelaudochukwuka**

### CODE REVIEW

File: src/html/svg/languages/bootstrap.html [hunk 1]
```diff
@@ -1,6 +1 @@
...
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/bootstrap.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/bootstrap.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/bulma.html [hunk 2]
```diff
@@ -1,5 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="120" height="120" viewBox="-58.5 0 373 373" class="position-relative">
-  <g>
-    <polygon fill="#00D1B2" fill-rule="nonzero" points="0 256 23.2727273 93.0909091 116.363636 0 232.727273 116.363636 162.909091 186.181818 256 279.272727 116.363636 372.363636" />
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/bulma.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/bulma.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/css3.html [hunk 3]
```diff
@@ -1,15 +1 @@
...
-          </g>
-          <path d="M383.824, 157.206l-4.801, 54.098l-14.5, 162.185l-108.172, 29.994l-0.094, 0.027l-108.119-30.021 l-7.541-84.817h52.968l3.938,43.992l58.785, 15.908l0.029-0.027l58.875-15.912l8.023-73.3l-185.154, 0.512l-5.251-50.125 l194.844-2.306l3.943-53.648l-204.005, 0.572l-3.43-49.913h264.369L383.824,157.206z M383.824, 157.206" style="fill:#FCFCFC;" />
-        </g>
-      </g>
-    </g>
-  </g>
-  <g id="Layer_1" />
-</svg>
\ No newline at end of file
+<img src="./images/languages/css3.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/css3.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/docker.html [hunk 4]
```diff
@@ -1,19 +1 @@
...
-        <path d="M476.441, 219.095c-10.35-6.972-34.21-9.487-52.537-6.036 c-2.371-17.249-12.003-32.27-29.538-45.782l-10.062-6.685l-6.685, 10.062c-13.225, 19.98-16.817, 52.896-2.658, 74.602 c-6.252, 3.377-18.545, 7.978-34.786, 7.689H27.757c-6.252, 36.511, 4.169,83.945, 31.624,116.501 c26.663, 31.551, 66.623, 47.579,118.872, 47.579c113.124, 0,196.851-52.104, 236.021-146.76c15.381, 0.288, 48.584, 0.073, 65.618-32.483 c1.078-1.797, 4.743-9.488,6.107-12.291L476.441,219.095z" style="fill: #099CEC;" />
-        <polygon points="109.186, 199.043 109.113, 199.043 61.68, 199.043 61.68, 241.734 109.186, 241.734" style="fill: #099CEC;"/>
-        <rect height="42.691" style="fill: #099CEC;" width="47.505" x="117.738" y="199.043" />
-      </g>
-    </g>
-  </g>
-  <g id="Layer_1" />
-</svg>
\ No newline at end of file
+<img src="./images/languages/docker.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/docker.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/gatsby.html [hunk 5]
```diff
@@ -1,4 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="130" height="130" viewBox="0 0 32 32" class="position-relative">
-  <path d="M27, 16H20v2h4.8A9.066, 9.066, 0, 0, 1, 19, 24.5L7.5, 13A9, 9, 0, 0, 1, 16, 7a9.232, 9.232, 0, 0, 1, 7.4, 3.8l1.5-1.3A10.993, 10.993, 0, 0, 0, 5.3, 13.6L18.5, 26.8A11.251, 11.251, 0, 0, 0, 27, 16ZM5, 16.1a10.682, 10.682, 0, 0, 0, 3.2, 7.6, 10.855, 10.855, 0, 0, 0, 7.6, 3.2Z" style="fill: #fff"/>
-  <path d="M16, 2A14, 14, 0, 1, 0, 30, 16, 14.041, 14.041, 0, 0, 0, 16, 2ZM8.2, 23.8A10.855, 10.855, 0, 0, 1, 5, 16.2L15.9, 27A11.351, 11.351, 0, 0, 1, 8.2, 23.8Zm10.2, 2.9L5.3, 13.6A10.993, 10.993, 0, 0, 1, 24.9, 9.5l-1.5, 1.3A9.232, 9.232, 0, 0, 0, 16, 7a9.1, 9.1, 0, 0, 0-8.5, 6L19, 24.5A9.066, 9.066, 0, 0, 0, 24.8, 18H20V16h7A11.031, 11.031, 0, 0, 1, 18.4, 26.7Z" style="fill: #639" />
-</svg>
\ No newline at end of file
+<img src="./images/languages/gatsby.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/gatsby.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/graphql.html [hunk 6]
```diff
@@ -1,5 +1 @@
...
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/graphql.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/graphql.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/gulp.html [hunk 7]
```diff
@@ -1,9 +1 @@
...
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/gulp.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/gulp.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/html5.html [hunk 8]
```diff
@@ -1,15 +1 @@
...
-          </g>
-          <path d="M382.656, 157.206H182.531l4.773, 54.098h190.584l-14.348, 162.21L256.352, 403.51l-0.094, 0.033 l-107.129-30.029l-7.485-84.843h52.49l3.91,44.02l58.246, 15.881l0.063-0.027l58.309-15.881l6.082-68.512H139.497 l-14.143-159.727h261.96L382.656, 157.206z M382.656, 157.206" style="fill: #FCFCFC;" />
-        </g>
-      </g>
-    </g>
-  </g>
-  <g id="Layer_1" />
-</svg>
\ No newline at end of file
+<img src="./images/languages/html5.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/html5.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/javascript.html [hunk 9]
```diff
@@ -1,7 +1 @@
...
-    <path d="M152.380952, 211.354413 L171.969422, 200.0128 C177.125994, 208.433981 183.827911, 214.619835 195.684368, 214.619835 C205.652521, 214.619835 212.009041, 209.635962 212.009041, 202.762159 C212.009041, 194.513676 205.479416, 191.592025 194.481168, 186.78207 L188.468419, 184.202565 C171.111213, 176.81473 159.597308, 167.53534 159.597308, 147.944838 C159.597308, 129.901308 173.344508, 116.153295 194.825752, 116.153295 C210.119924, 116.153295 221.117765, 121.48094 229.021663, 135.400432 L210.29059, 147.428775 C206.166146, 140.040127 201.699556, 137.119289 194.826159, 137.119289 C187.78047, 137.119289 183.312254, 141.587098 183.312254, 147.428775 C183.312254, 154.646349 187.78047, 157.568406 198.089956, 162.036622 L204.103924, 164.614095 C224.553448, 173.378641 236.067352, 182.313448 236.067352, 202.418387 C236.067352, 224.071924 219.055137, 235.927975 196.200432, 235.927975 C173.860978, 235.927975 159.425829, 225.274311 152.381359, 211.354413" fill="#000000" />
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/javascript.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/javascript.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/jquery.html [hunk 10]
```diff
@@ -1,7 +1 @@
...
-  <path d="m2.436 17.992-.004-.006-.007-.009zm4.158-.744.01.007.002.003z" />
-  <path d="m13.666 1.427c-.909 1.327-.996 2.975-.37 4.438.665 1.554 2.024 2.774 3.608 3.352.065.025.128.046.195.07l.088.027c.092.029.185.063.28.084 4.381.841 5.567-2.238 5.886-2.691-1.043 1.49-2.792 1.848-4.938 1.329a4.753 4.753 0 0 1 -.55-.17l.033.011a6.196 6.196 0 0 1 -.656-.27l.038.017c-.41-.2-.762-.416-1.089-.664-1.918-1.45-3.104-4.211-1.852-6.46z" fill="#7ACEF4" />
-</svg>
\ No newline at end of file
+<img src="./images/languages/jquery.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/jquery.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/react.html [hunk 11]
```diff
@@ -1,12 +1 @@
...
-  </g>
-  <g>
-    <path class="st0" d="M413.4, 550.1c-27.2, 0-61.7-16.4-97.7-47.4c-40.2-34.6-81.1-86.1-115.3-145.2v0c-33.6-58-57.6-118.3-67.7-170 c-5.9-30.2-7-57.9-3.2-80.2c4.2-24.3, 14.1-41.6, 28.8-50.1c29.8-17.3, 80.5, 0.5, 132.1, 46.4c36.6, 32.5, 75, 81.9, 108.1, 139.1 c35.3, 61, 59, 118.5, 68.4, 166.3c6.1, 31, 7.1, 59.8, 2.8, 83.2c-4.6, 24.9-15, 42.6-30, 51.3C432.2, 547.9, 423.3, 550.1, 413.4, 550.1z M221.2, 345.5c32.4, 56, 72.6, 106.7, 110.2, 139c45.1, 38.8, 80.9, 47.2, 96.4, 38.2c16.1-9.3, 27.9-47.4, 15.7-109 c-9-45.2-31.7-100.2-65.7-158.9c-31.9-55.1-68.6-102.4-103.3-133.2C225.9, 78.4, 186.5, 68.7, 170.4, 78c-8.2, 4.7-14.3, 16.6-17.2, 33.4 c-3.3, 19.4-2.3, 44.2, 3.1, 71.5C165.9, 232, 188.9, 289.7, 221.2, 345.5L221.2, 345.5z" fill="#00D8FF" />
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/react.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/react.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/rwd.html [hunk 12]
```diff
@@ -1,10 +1 @@
...
-    <path fill-rule="evenodd" fill="#6EC9C2" d="M359.862, 243.696c45.552, 0,91.104, 0, 138.885, 0c-2.839, 28.678-5.448, 55.6-8.212, 82.509 c-0.729, 7.101-5.955, 9.032-12.061, 9.047c-77.356, 0.178-154.711, 0.315-232.061, 0.285c-1.717, 0-3.426-2.037-5.138-3.126 c38.859,0,77.718, 0, 118.586, 0C359.862,300.609,359.862, 272.152,359.862, 243.696z" />
-    <path fill-rule="evenodd" fill="#21A691" d="M359.862, 243.696c0, 28.456, 0,56.913, 0, 88.715c-40.868, 0-79.727, 0-118.586, 0 l-0.003-0.157c0.132-7.394, 2.393-15.082-2.983-21.719c0-21.672, 0-43.339,0-66.839C277.263, 243.696, 318.562, 243.696, 359.862, 243.696 z" />
-    <path fill-rule="evenodd" fill="#6EC9C2" d="M238.289, 310.536c5.376, 6.637, 3.116, 14.325, 2.983, 21.719 C235.074, 325.73,237.79, 317.98, 238.289, 310.536z" />
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/rwd.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/rwd.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/sass.html [hunk 13]
```diff
@@ -1,10 +1 @@
...
-      </g>
-    </g>
-  </g>
-  <g id="Layer_1" />
-</svg>
\ No newline at end of file
+<img src="./images/languages/sass.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/sass.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/typescript.html [hunk 14]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<img src="./images/languages/typescript.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/typescript.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/vue.html [hunk 15]
```diff
@@ -1,7 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="130" height="130" viewBox="0 -17.5 256 256" class="position-relative">
-  <g>
-    <path d="M204.8, 0 L256, 0 L128, 220.8 L0, 0 L50.56, 0 L97.92, 0 L128, 51.2 L157.44, 0 L204.8, 0 Z" fill="#41B883" />
-    <path d="M0, 0 L128,220.8 L256, 0 L204.8,0 L128, 132.48 L50.56, 0 L0, 0 Z" fill="#41B883" />
-    <path d="M50.56, 0 L128,133.12 L204.8, 0 L157.44, 0 L128, 51.2 L97.92, 0 L50.56, 0 Z" fill="#35495E" />
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/vue.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/vue.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/webpack.html [hunk 16]
```diff
@@ -1,7 +1 @@
...
-    <path d="M123.363019, 214.846792 L61.3433962, 180.697358 L61.3433962, 113.123019 L123.363019, 148.914717 L123.363019, 214.846792 L123.363019, 214.846792 Z M132.250566, 214.846792 L194.270189, 180.74566 L194.270189, 113.123019 L132.250566, 148.914717 L132.250566, 214.846792 Z M65.4973585, 105.298113 L127.806792, 71.0520755 L190.067925, 105.298113 L127.806792, 141.234717 L65.4973585, 105.298113 L65.4973585, 105.298113 Z" fill="#1C78C0" />
-  </g>
-</svg>
\ No newline at end of file
+<img src="./images/languages/webpack.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/webpack.html:1]: Could you wrap this <picture> tag

File: src/html/svg/languages/wordpress.html [hunk 17]
```diff
@@ -1,3 +1 @@
...
-</svg>
\ No newline at end of file
+<img src="./images/languages/wordpress.svg" alt="icon" class="position-relative" width="50px" height="50px">
```
Comment (RejaurRahman) [inline src/html/svg/languages/wordpress.html:1]: Could you wrap this <picture> tag

Comment (Onyelaudochukwuka) [reply src/html/svg/languages/wordpress.html:1]: It's wrapped on render so wrapping this here will result in unnecessary DOM size

Comment (Onyelaudochukwuka) [reply src/html/svg/languages/wordpress.html:1]: So for performance sake it's optimal to keep your DOM clean and short where possible

Comment (Onyelaudochukwuka) [reply src/html/svg/languages/wordpress.html:1]: but if you still want me to I'll.
do you?

**Pull request closed without merging**